files/en-us/web/api/csspropertyrule/initialvalue/index.html

```diff
@@ -0,0 +1,52 @@
+---
+title: CSSPropertyRule.initialValue
+slug: Web/API/CSSPropertyRule/initialvalue
+tags:
+  - API
+  - CSS
+  - CSS Properties and Values API
+  - CSSPropertyRule
+  - Experimental
+  - Houdini
+  - Property
+  - Reference
+  - Read-only
+---
+<div>{{APIRef("CSS Properties and Values API")}}{{SeeCompatTable}}{{draft}}</div>
+
+<p class="summary">The read-only <strong><code>initialValue</code></strong> null property of the {{domxref("CSSPropertyRule")}} interface returns the initial value of the custom property registration represented by the {{domxref("@property")}} rule, controlling the property’s initial value. </p>
```

Comment:
```suggestion
<p class="summary">The read-only <strong><code>initialValue</code></strong> nullable property of the {{domxref("CSSPropertyRule")}} interface returns the initial value of the custom property registration represented by the {{domxref("@property")}} rule, controlling the property’s initial value. </p>
```
*nullable* rather than *null*?

Reply:
this was an error left in from the generator.